contracts/interfaces/gov/IGovPool.sol

```diff
@@ -155,4 +163,9 @@ interface IGovPool {
         external
         view
         returns (uint256, ShrinkableArray.UintArray memory);
+
+    function getProposals(uint256 offset, uint256 limit)
```

Comment:
Let's have the order of functions in the interface match the order in the contract

Reply:
done